doc/_static/style.css

```diff
@@ -167,8 +198,90 @@ dt:target code {
 /* Display content (e.g., images) in the middle center of a div */
 .midcenter {
     display: table-cell;
-    vertical-align:middle;
+    vertical-align: middle;
     text-align: center;
     float: none;
-    height: 280px;
-}
\ No newline at end of file
+}
+.tall {
+    height: 250px;
+}
+
+/* our carousel */
+.chopper {
+    overflow: hidden;
+    height: 250px;
+}
+.carousel-image {
+    position: absolute;
+    width: 100%;
+    opacity: 1.0;
+    left: 50%;
+    top: 50%;
+    transform: translateY(-50%) translateX(-50%);
+}
+
+.cont{
+    width: 100%;
+    display: table;
+    border-spacing: 20px 5px;
+}
+.cont:before {
+    content: "";
+}
+.cont:after {
+    content: "";
+}
+.btn-cont {
+    display: table-cell;
+    width: 25%;
+}
+.footer {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+}
+.footer a {
+    font-weight: 700;
+}
+.jumbotron {
+    background-color: #fff;
+}
+.support-front {
+    font-size: 10px;
+}
+.carousel-caption {
+    text-shadow: 0 1px 1px rgba(0,0,0,1.0);
+    background-color: rgba(0,0,0,0.5);
+    box-shadow: 0 0 5px rgba(0,0,0,0.5);
+}
+.carousel-caption p {
+    font-size: 16px;
+}
+.tagline {
+  font-size: 24px !important;
```

Comment:
can you use:

.tagline {
  font-size: 22px !important;
  text-align: center;
}